src/netron_output.cpp

```diff
@@ -151,17 +158,17 @@ auto make_onnx_json_literal(instruction_ref ins,
 auto make_onnx_json_shape(const shape& s)
 {
     value ret;
-    value dim;
-    auto shape_lens = s.lens();
-    std::transform(shape_lens.begin(),
-                   shape_lens.end(),
-                   std::back_inserter(dim),
-                   [](std::size_t len) { return len; });
+    value dim = value({});
```

Comment:
Actually `value::object{}` would be cleaner here.

Reply:
Yes, didn't realize that's how it worked.